pkg/build/generator/generator.go

```diff
@@ -501,8 +502,8 @@ func (g *BuildGenerator) generateBuildFromConfig(ctx apirequest.Context, bc *bui
 			Labels: bcCopy.Labels,
 			OwnerReferences: []metav1.OwnerReference{
 				{
-					APIVersion: "v1",          // BuildConfig.APIVersion is not populated
-					Kind:       "BuildConfig", // BuildConfig.Kind is not populated
+					APIVersion: buildapiv1.SchemeGroupVersion.String(), // BuildConfig.APIVersion is not populated
```

Comment:
@mfojtik @deads2k ok this is my last try.....  is this right?  (i'm rejecting the variable suggestion because i'd have to create the variable really far away from this code)